src/Dialect/ONNX/Transforms/QDQAroundOpOpt.cpp

```diff
@@ -0,0 +1,132 @@
+//===- QDQOpt.cpp - Remove QDQ operations --------*- C++ -*-===//
+//
+// (c) Copyright 2022 - 2025 Advanced Micro Devices, Inc. All Rights Reserved.
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir/Transforms/DialectConversion.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+#include <cmath>
+#include <mlir/IR/IRMapping.h>
+#include <mlir/IR/Operation.h>
+#include <mlir/IR/PatternMatch.h>
+#include <mlir/Pass/Pass.h>
+#include <src/Dialect/ONNX/ONNXOps.hpp>
+#include <src/Dialect/ONNX/ONNXOps/OpHelper.hpp>
```

Comment:
nit:
```suggestion
#include <src/Dialect/ONNX/ONNXOps.hpp>
#include <src/Dialect/ONNX/ONNXOps/OpHelper.hpp>

#include <mlir/Transforms/DialectConversion.h>
#include <mlir/Transforms/GreedyPatternRewriteDriver.h>
#include <mlir/IR/IRMapping.h>
#include <mlir/IR/Operation.h>
#include <mlir/IR/PatternMatch.h>
#include <mlir/Pass/Pass.h>

#include <cmath>

```
We generally follow the LLVM style guide: https://llvm.org/docs/CodingStandards.html#include-style (although onnx-mlir is not so strict with it)